src/Microsoft.OpenApi.OData.Reader/Microsoft.OpenAPI.OData.Reader.csproj

```diff
@@ -12,7 +12,7 @@
     <Company>Microsoft</Company>
     <Title>Microsoft Open API OData Reader</Title>
     <RootNamespace>Microsoft.OpenApi.OData</RootNamespace>
```

Comment:
Just checking here, does this mean we want to stop supporting runtimes like net framework? 
Anyone pulling the package from a net framework project or version earlier than net8.0 won't be able to use this and could arguably be a breaking change....

Reply:
We may need to bump the major version here.

Reply:
We have bumped the major version.